extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -908,6 +906,12 @@ export class VisibilityManagerForDoc extends VisibilityManager {
     intersectionRatio = Math.min(Math.max(intersectionRatio, 0), 1);
     const id = getElementId(target);
     const trackedElement = this.trackedElements_[id];
+    if (boundingClientRect.width < 1 || boundingClientRect.height < 1) {
```

Comment:
should this be collapsed with the declaration above?

```javascript
if (width<1 || height<1) {
  // Set intersectionRatio to 0 when the element is not visible.
  // Use < 1 because the width/height can be a double values.
  intersectionRatio = 0;
} else {
  intersectionRatio = Math.min(Math.max(intersectionRatio, 0), 1);
}
```

Reply:
done